src/node/entities.h

```diff
@@ -44,26 +44,27 @@ namespace ccf
 
   struct Tables
   {
-    static constexpr auto MEMBERS = "members";
-    static constexpr auto MEMBER_ACKS = "memberacks";
-    static constexpr auto MEMBER_CERTS = "membercerts";
-    static constexpr auto USER_CERTS = "usercerts";
-    static constexpr auto NODE_CERTS = "nodecerts";
-    static constexpr auto NODES = "nodes";
-    static constexpr auto VALUES = "values";
-    static constexpr auto APP = "app";
-    static constexpr auto APP_PUBLIC = "app_public";
-    static constexpr auto SIGNATURES = "signatures";
-    static constexpr auto USER_CLIENT_SIGNATURES = "userclientsignatures";
-    static constexpr auto MEMBER_CLIENT_SIGNATURES = "memberclientsignatures";
-    static constexpr auto WHITELISTS = "whitelists";
-    static constexpr auto PROPOSALS = "proposals";
-    static constexpr auto GOV_SCRIPTS = "govscripts";
-    static constexpr auto APP_SCRIPTS = "appscripts";
-    static constexpr auto SECRETS = "secrets";
-    static constexpr auto CODEID = "codeid";
-    static constexpr auto VOTING_HISTORY = "votinghistory";
-    static constexpr auto SERVICE = "service";
+    static constexpr auto MEMBERS = "ccf.members";
+    static constexpr auto MEMBER_ACKS = "ccf.member_acks";
+    static constexpr auto MEMBER_CERTS = "ccf.member_certs";
+    static constexpr auto USER_CERTS = "ccf.user_certs";
+    static constexpr auto NODE_CERTS = "ccf.node_certs";
+    static constexpr auto NODES = "ccf.nodes";
+    static constexpr auto VALUES = "ccf.values";
+    static constexpr auto APP = "ccf.app";
+    static constexpr auto APP_PUBLIC = "ccf.app_public";
+    static constexpr auto SIGNATURES = "ccf.signatures";
+    static constexpr auto USER_CLIENT_SIGNATURES = "ccf.user_client_signatures";
+    static constexpr auto MEMBER_CLIENT_SIGNATURES =
+      "ccf.member_client_signatures";
+    static constexpr auto WHITELISTS = "ccf.whitelists";
+    static constexpr auto PROPOSALS = "ccf.proposals";
+    static constexpr auto GOV_SCRIPTS = "ccf.gov_scripts";
+    static constexpr auto APP_SCRIPTS = "ccf.app_scripts";
+    static constexpr auto SECRETS = "ccf.secrets";
+    static constexpr auto CODEID = "ccf.code_id";
```

Comment:
While we're here:
- For consistency, this should be `CODE_IDS` and `ccf.code_ids`
- Do we still need the `APP` and `APP_PUBLIC` tables? The logging app uses these, but I believe the apps should name their own tables (app-writer defines the types, and usually wants multiple tables - they should name them too)

Reply:
1. Yes, done
2. Checking, good point!

Reply:
So about 2. the only difficulty is that we need to whitelist them as USER_APP_CAN_WRITE, so until we have made the whitelist configurable, we can't do away with them. Opened as #355

Reply:
I think this whitelist is already app-modifiable, and we already have `pub0`, `priv5` etc tables in the `luageneric` app for this purpose?

Reply:
So that's true, although we're still missing a command line mechanism to set that, so #355 still stands I think. The tables in luageneric are GenericTables (json to json) rather than Scripts. Scripts are App/App Public/Gov, and they are currently a little special, and not app-defined tables.

The logging app is writing its data to pub0/priv0, these are just for code. I'm reading this again, and failing to understand what's suggested now. Are you saying tables should define (and wire) their own code tables?

Reply:
Ok, I think I know what you mean. The only problem with this I think is that unless the app table is hardcoded, the app needs to tell either the startup arguments (now), or the initial member vote( once we've done #293) where to install the scripts.

Reply:
Sorry, getting confused between APP and APP_SCRIPTS, ignore this, you are right, APP and APP_PUBLIC can be removed.